gateway-developer-plugin/src/main/java/com/ca/apim/gateway/cagatewayconfig/BuildFullBundleTask.java

```diff
@@ -112,7 +115,11 @@ public void perform() {
 
             final Pair<String, Map<String, String>> bundleEnvironmentValues = environmentConfigurationUtils.parseBundleMetadata(metaDataFile, configuredFolder);
             if (null != bundleEnvironmentValues) {
-                final String fullInstallBundleFilename = bundleEnvironmentValues.getLeft() + FULL_INSTALL_BUNDLE_NAME_SUFFIX;
+                String fullInstallBundleFilename = bundleEnvironmentValues.getLeft();
+                if (StringUtils.isNotBlank(projectInfo.getVersion())) {
+                    fullInstallBundleFilename = fullInstallBundleFilename + PREFIX_FULL;
```

Comment:
This should have version as well in the name?

Reply:
version is already part of the file name